history/cultures/twilights_hammer.txt

```diff
@@ -23,6 +23,7 @@
 	discover_innovation = innovation_ledger 
 	discover_innovation = innovation_crop_rotation 
 	#
+	discover_innovation = innovation_war_camels
```

Comment:
Other cultures that should have camels, don't have them.

Reply:
I gave it to tol'vir, who else should have it?

Reply:
> I gave it to tol'vir, who else should have it?

Everyone who lives in that region.

Reply:
woops, i thought i already gave it to the rest